### ISSUE TITLE
Support for private messages over ActivityPub

### ISSUE THREAD

**Issue by thomasjwebb** [akirk/friends#417]
If I send a message from Mastodon to my blog, I cannot see it from my blog. And I do not have the ability to send private messages from my blog to Fediverse accounts. It seems this only supports messaging between WordPress installations with this plugin installed.

**Comment by akirk** [comment]
Yes, currently it's not supported but certainly something we can implement. Since there are already [direct messages between Friends](https://github.com/akirk/friends/blob/main/includes/class-messages.php), adding support for sending those via ActivityPub should be not too hard, I hope to get to this in the coming weeks.

**Comment by akirk** [comment]
One more usecase listed here: https://wordpress.org/support/topic/direct-messages-via-activitypub/ sending a message to BridgyFed.
